### PR TITLE
Ballot 169

### PR DESCRIPTION
**Pull request by pzb**
These are the changes from the endorsed (but not passed) Ballot 169.

### CODE REVIEW

File: docs/BR.md [hunk 1]
```diff
@@ -280,6 +290,22 @@ No stipulation.
...
+
+A Request Token MAY include other information to ensure its uniqueness.
+
+A Request Token that includes a timestamp SHALL remain valid for no more than 30 days from the time of creation.
+
+A Request Token that includes a timestamp SHALL be treated as invalid if its timestamp is in the future.
+
+A Request Token that does not include a timestamp is valid for a single use and the CA SHALL NOT re-use it for a subsequent validation.
+
+The binding SHALL use a digital signature algorithm or a cryptographic hash algorithm at least as stro
```
Comment (jsha) [inline docs/BR.md]: This line cuts off at the end.

**Comment by mdssc** [comment]
anyone can point me where those vertical colored lines in the text window have been explained? Thanks.

**Comment by pzb** [comment]
@mdssc Green means the section is new, Red means the section was removed, and Orange means it was changed and the changes are shown inline.

**Comment by mdssc** [comment]
ok, its clear now. Many thanks!

M.D.

**Comment by pzb** [comment]
I'm killing this pull request in favor of a new one that has a squashed history

**Pull request closed without merging**